src/services/FilterService.ts

```diff
@@ -0,0 +1,33 @@
+import { Filters } from '../types/Filters/Filters';
+import { Todo } from '../types/Todo/Todo';
+
+export const filterTodos = (todos: Todo[], isCompleted: boolean) => {
+  return todos.filter(todo => todo.completed === isCompleted);
+};
+
+export const getActiveTodos = (todos: Todo[]) => {
+  return filterTodos(todos, false);
+};
+
+export const getCompletedTodos = (todos: Todo[]) => {
+  return filterTodos(todos, true);
+};
+
+export const isAllTodosComplete = (todos: Todo[]) => {
+  return todos.every(todo => todo.completed === true);
+};
+
+export const getOneCompletedTodo = (todos: Todo[]) => {
```

Comment:
```suggestion
export const checkHasCompletedTodo = (todos: Todo[]) => {
```
functions with `get` should return something - if `getOneCompletedTodo` it should return todo